packages/jest-runtime/src/index.ts

```diff
@@ -1172,8 +1187,28 @@ class Runtime {
       });
     };
 
+    const runtime = this;
+
     // should we implement the class ourselves?
-    class Module extends nativeModule.Module {}
+    class Module extends nativeModule.Module {
+      _compile(content: string, filename: string) {
```

Comment:
```suggestion
      _compile(content: unknown, filename: unknown) {
```

I believe your `typeof`s narrow the type correctly.

Reply:
I'm not really familiar with Typescript - I'm guessing that it'll infer the type based on type guards? This feels like a convention I'm not aware of - wouldn't we rather have the types be explicit for readability?

Reply:
it's more honest since we get it from user input. And the `typeof`s will narrow it to `string`